File: Server/src/main/java/com/springboot/garageApp/GarageAppApplication.java
package com.springboot.garageApp;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.domain.EntityScan;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@SpringBootApplication
@ComponentScan(basePackages = "com.springboot.garageApp.*")
@EntityScan(basePackages = "com.springboot.garageApp.*")
@EnableJpaRepositories(basePackages = "com.springboot.garageApp.*")
public class GarageAppApplication {

	public static void main(String[] args) {
		SpringApplication.run(GarageAppApplication.class, args);
	}

}

File: Server/src/main/java/com/springboot/garageApp/GarageAppApplication.kt
package com.springboot.garageApp

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
object GarageAppApplication {
    @JvmStatic
    fun main(args: Array<String>) {
        SpringApplication.run(GarageAppApplication::class.java, *args)
    }
}
